src/drive/web/modules/viewer/helpers.js

```diff
@@ -1,11 +1,13 @@
 import has from 'lodash/has'
+import get from 'lodash/get'
 
 export const hasCertifications = ({ file }) =>
   has(file, 'metadata.carbonCopy') || has(file, 'metadata.electronicSafe')
 
-export const showPanel = ({ file }) => {
-  // TODO add rules for connectors
+export const isFromConnector = ({ file }) =>
+  get(file, 'cozyMetadata.uploadedBy.slug') !== 'drive' &&
+  get(file, 'cozyMetadata.uploadedBy.slug') !== 'cozy-desktop'
```

Comment:
I think we can find something better. Soon we will be able to upload a photo from cozy-notes, resulting of having a wrong behavior on our viewer. 

@doubleface do we always have sourceAccount or sourceAccountIdentifier in a file imported by a konnector? Can we make the difference between konnector and app this way?

Reply:
Aux dernières nouvelles et sans changement on ne peut pas faire autrement, j'avais demandé à @doubleface avant la PR, ainsi qu'à @taratatach pour le client desktop. Je te rejoins sur le fait que c'est pas une solution ultime ici...

Reply:
Je reste convaincu qu'on aura le sourceAccount ou le sourceAccountIdentifier dans les meta d'un fichier importé par un connecteur, même si le connecteur est désinstallé ou autre. C'est plutôt sur cette info qu'on devrait se baser. Impossible qu'on se base sur le Slug, vu que le slug peut aussi changer en fonction des surcharges. 

Je cite https://github.com/cozy/cozy-doctypes/blob/master/docs/README.md#document-metadata 

```
sourceAccount: When the document was imported from a connector, identifier of the account in io.cozy.accounts
sourceAccountIdentifier: When the document was imported from a connector, identifier of the account on targeted web service (the email address most of the time)
```

ça semble vraiment faire ce que l'on veut

Reply:
Oui partons sur le fait d'avoir ou non le `sourceAccount`